sass/view3d-ar.sass

```diff
@@ -51,9 +51,130 @@
     .view3d-tooltip
       visibility: visible
 
+.view3d-ar-root
```

Comment:
If you look at the screenshots in the description, the clock modeling is small or out of focus.

The ar detection icon does not seem to be visible.

Reply:
That screenshot is just for showing how it looks for reviewers, the icon will be shown above the camera feed so that won't be matter.